### DISCUSSION TITLE
Zoom cask upgrade

### DISCUSSION THREAD

**Discussion by sosterberg** [Homebrew/discussions#626] [Casks] [answered]
Hi,

Just installed zoom cask using homebrew + Rosetta 2. After install Zoom suggests that I update to their version compatible with M1 apple computers. 

How does this go with my Rosetta 2 installation? I.e. will it "break/mess up" the homebrew installation?

**Comment by Linus-XZX** [comment, marked as answer]
Don't think so but this is not the best way. Cask `zoom` don't have `auto_update` or `version_latest` (intentional according to https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/zoom.rb), so `brew upgrade` will upgrade it even if you have upgraded with Zoom's own mechanism, effectively leading to a reinstall. And this kind of switching between architectures doesn't sound so good.
If you have a ARMed Homebrew, I personally believe directly using the ARM version will be a better idea.
